contents/newsletter/activation-metrics.md

```diff
@@ -0,0 +1,156 @@
+
+## What makes activation tricky?
+
+If you look around for how to define your activation metric, you'll find a bunch of different advice. Common ones include it being your "aha" moment or when you make your users happy. Why isn't there a simple, shared definition?
+
+1. **It's unique to your product.** There's no standard way to define activation (this is also why benchmarks suck).[^1]
+
+2. **It could be many things.** There are likely many things you want users to do in your product: complete onboarding, upload a file, share a link, watch a video. Activation could mean doing any of these (or a combination of them).
+
+3. **Activating may mean doing something multiple times.** Adding to the complexity is that someone might need to do something multiple times in order to get it. For example, to activate into our session replay product means analyzing not one but five replays. 
+
+![Replay](https://res.cloudinary.com/dmukukwp6/image/upload/replay_898e279f51.png)
+
+## How we find activation metrics at PostHog
+
+At PostHog, we have a lot of experience [finding our activation metric](/product-engineers/activation-metrics). This is because we have 8+ products, each requiring their own activation metric. This metric could be some combination of 10+ events that are all important for the product.
+
+Our process starts by gathering a big list of events we think might "hook" users into our product. We get these by looking at usage of that specific product and by asking customer-facing teams what they think.
+
+![Message](https://res.cloudinary.com/dmukukwp6/image/upload/message_4119900c6c.png)
+
+Next, we create groups of 3-5 events to test together. We aimed for 5-10 different groups, including the number of times the event occurs (e.g. watched 5 replays vs. watched 1 replay).
+
+![Groups](https://res.cloudinary.com/dmukukwp6/image/upload/image_16_bf16064d65.png)
+
+With our potential activation event groups, we write a query to test how they correlate with retention:
+
+1. Get companies that start with our product broken down by month. We use group analytics for this, but you could just do users too.
+
+2. Filter for companies who completed the activation event group in the required time. We require companies to activate within 30 days of signing up. 
+
+3. Calculate retention for successful companies. We check to see how many of them are still using the product 3 months after starting.
+
+![Activation](https://res.cloudinary.com/dmukukwp6/image/upload/image_17_a3f3ac0bfe.png)
+
+We then compare the retention against our product's average retention rate for all companies. We look for a retention percentage that's nominally higher than our average retention rate. 
+
+The above example has a retention rate of 66% for activated companies. This means:
```

Comment:
I'm unsure what example this is referring to here

Reply:
It's the image above, making this clearer